Content.Client/Wall/ReinforcedWallVisualsComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Client.Wall;
+
+public sealed class ReinforcedWallVisualsComponent : Component
```

Comment:
you need to add `[RegisterComponent]` as an attribute here - that might be why your tests are failing